include/ccl_neutrinos.h

```diff
@@ -31,15 +31,6 @@ typedef enum ccl_neutrino_mass_splits{
   ccl_nu_single=4
 } ccl_neutrino_mass_splits;
 
-/**
- * Spline for the phasespace integral required for getting the fractional energy density of massive neutrinos.
- * Returns a gsl spline for the phase space integral needed for massive neutrinos.
- * @param status Status flag. 0 if there are no errors, nonzero otherwise.
- * For specific cases see documentation for ccl_error.c
- * @return spl, the gsl spline for the phasespace integral required for massive neutrino calculations.
- */
-gsl_spline* calculate_nu_phasespace_spline(int *status);
```

Comment:
Note for self: Why do we not need this anymore?

Reply:
This function is not used anywhere else, so I made it static and removed it from the `.h` file.